src/perspective_view/render_layer.ts

```diff
@@ -50,6 +50,11 @@ export interface PerspectiveViewRenderContext
    * Specifies whether there was a previous pick ID pass.
    */
   alreadyEmittedPickID: boolean;
+
+  /**
+   * Specifies the ID of the depth frame buffer texture to query during rendering.
+   */
+  depthBufferTextureID?: WebGLTexture | null;
```

Comment:
Since `WebGLTexture` is a garbage-collected object rather than a numeric identifier, it may make sense to just call this `depthBufferTexture` instead.